firefox/harmful.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE HTML>
+<html lang="en-US" dir="ltr">
+<head>
+    <meta charset="utf-8">
+
+    <title>Unwanted!</title>
+    <link href="/css/min.css" rel="stylesheet">
+    <link rel="stylesheet" href="/css/its-a-trap.css" media="screen" />
+</head>
+
+<body id="unwanted" class="portal-page locale-en-US portal-page">
+
+<div id="wrapper">
+
+<div id="doc">
+
+    <div id="nav-access">
+        <a href="#nav-main">skip to Navigation</a>
+        <a href="#switch">switch language</a>
+    </div>
+
+    <!-- start #header -->
+    <div id="header">
+        <div>
+        <h1><a href="http://www.mozilla.org/firefox/" title="Back to home page">Mozilla Firefox</a></h1>
+        <a href="http://www.mozilla.org/" class="mozilla">visit <span>mozilla</span></a>
+
+        </div>
+    </div>
+    <!-- end #header -->
+
+
+<div id="main-feature">
+    <h2>Harmful!</h2>
+    <p>I’m a naughty Web page trying to install dangerous apps!</p>
+</div>
+
+<div id="main-content">
+
+<p>OK, not really, but some sites can offer apps that steal or delete your information (for example, photos, passwords, messages, and credit cards). Firefox provides built-in <a href="http://www.mozilla.org/firefox/phishing-protection/">Phishing and Malware Protection</a> to help you avoid potentially harmful applications. If you are using Firefox 40, Firefox for Android 60 or later, you should have been warned to stay away from this page."</p>
```

Comment:
That should be "If you are using Firefox for Android 60 or later,".

The "Firefox 40," should not be there.